devops/github_issue_alert/program.py

```diff
@@ -0,0 +1,57 @@
+"""Monitors GitHub issue creation and comment activity.
+
+Sending the notifications to Slack to keep teams updated.
+"""
+
+import os
+
+from autokitteh.slack import slack_client
+
+
+SLACK_CHANNEL = os.getenv("SLACK_CHANNEL_NAME_OR_ID", "")
+
+slack = slack_client("slack_conn")
+
+
+def on_issue_comment(event):
+    """Processes a GitHub issue comment event and sends a message to Slack.
+
+     Supported actions:
+    - Creation
+    - Editing
+    - Deletion
```

Comment:
Uses less lines and uses names aligned with GitHub's terminology.
```suggestion
     Supported actions: created, edited, deleted
```